pysat/utils/io.py

```diff
@@ -0,0 +1,1033 @@
+                    step = len(data.variables[obj_var_keys[0]][0, :])
+
+                    # Check if there is an index we should use
+                    if not (index_key_name is None):
+                        time_var = loop_dict.pop(index_key_name)
+                        if time_index_flag:
+                            # Create datetime index from data
+                            time_var = pds.to_datetime(1.0E6 * time_var)
+                        new_index = time_var
+                        new_index_name = index_name
+                    else:
+                        # Using integer indexing if no index identified
+                        new_index = np.arange((loop_lim * step),
+                                              dtype=np.int64) % step
+                        new_index_name = 'index'
+
+                    # Load all data into frame
+                    if len(loop_dict.keys()) > 1:
+                        loop_frame = pds.DataFrame(loop_dict,
+                                                   columns=clean_var_keys)
+                        if obj_key in loop_frame:
+                            del loop_frame[obj_key]
+
+                        # Break massive frame into bunch of smaller frames
+                        for i in np.arange(loop_lim, dtype=np.int64):
+                            loop_list.append(loop_frame.iloc[(step * i):
+                                                             (step * (i + 1)),
+                                                             :])
+                            loop_list[-1].index = new_index[(step * i):
+                                                            (step * (i + 1))]
+                            loop_list[-1].index.name = new_index_name
+                    else:
+                        loop_frame = pds.Series(loop_dict[clean_var_keys[0]],
+                                                name=obj_var_keys[0])
+
+                        # Break massive series into bunch of smaller series
+                        for i in np.arange(loop_lim, dtype=np.int64):
+                            loop_list.append(loop_frame.iloc[(step * i):
+                                                             (step * (i + 1))])
+                            loop_list[-1].index = new_index[(step * i):
+                                                            (step * (i + 1))]
+                            loop_list[-1].index.name = new_index_name
+
+                    # Add 2D object data, all based on a unique dimension within
+                    # netCDF, to loaded data dictionary
+                    loaded_vars[obj_key] = loop_list
+                    del loop_list
+
+                # Prepare dataframe index for this netcdf file
+                time_var = loaded_vars.pop(epoch_name)
+
+                # Convert from GPS seconds to seconds used in pandas (unix time,
+                # no leap)
+                # time_var = convert_gps_to_unix_seconds(time_var)
```

Comment:
It looks like this is no longer being done?  @rstoneback @jklenzing

Reply:
I'm fine dropping this.  @rstoneback?

Reply:
I think this was already dropped while I was 'away.' Looked into the history and the commented line was never uncommented here. I think this section likely originated from the COSMIC project and this is a leftover. Thanks for the cleanup.